### PR TITLE
Adjust documentation for registration of catalog items

### PR DESCRIPTION
**Pull request by baptistepa**
1. Renamed the file for Looking_up_a _DataMiner_connector to Looking_up_a_catalog_item and updated its content
2. Added a file to explain how to register items to the catalog, both with the UI and with the API
3. Adjusted the file that explains how to manage your dcp keys to explain the different keys (system and organization)

### CODE REVIEW

File: user-guide/Cloud_Platform/Catalog/Looking_up_a_catalog_item.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+When you open up the Catalog module, a search box is displayed.
+
+1. In the search box, enter (part of) the name of the item you are looking for (e.g. a connector, a package, etc.).
+
+   As you type, a list with the top 5 results will be displayed under the search box.
+
+1. As soon as you notice the name of the item you are looking for, click it, and continue to the next step.
+
+   If you cannot find the item you are looking for, you can choose to get a list of all the search results by clicking the . This will open the browse page. On the left side of this page, you have filters that allow you to narrow down your search.
```
Comment (spetebroodt) [inline user-guide/Cloud_Platform/Catalog/Looking_up_a_catalog_item.md:15]: Something seems to be missing here.

File: user-guide/Cloud_Platform/Catalog/Looking_up_a_catalog_item.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+When you open up the Catalog module, a search box is displayed.
+
+1. In the search box, enter (part of) the name of the item you are looking for (e.g. a connector, a package, etc.).
+
+    As you type, a list with the top 5 results will be displayed under the search box.
+
+1. As soon as you notice the name of the item you are looking for, click it and continue to the next step.
+
+    If you cannot find the item you are looking for in this list you can choose to get a list of all the search results, by clicking the . This will open up the browse page. On the left side of this page you have filters that allow you to narrow down your search.
```
Comment (SimonV-Skyline) [inline user-guide/Cloud_Platform/Catalog/Looking_up_a_catalog_item.md]: "... by clicking the . This will open up..." There is probably a word missing there?

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+## Private catalog items
+
+All of the items that have been provided by Skyline Communications are considered to be public items. They are available for all our users. Note that for some of these items (eg Connectors) the user might need to buy a license to be able to use them on their system.
+
+Private catalog items are items that are only available to one or more organizations. This gives users the possibility to develop App Packages, Automation Scripts or any type of catalog item and redeploy them within their organization on various systems or share them with other organizations.
+
+## Versioning of catalog items
+
+Catalog items can hold multiple versions. To make sure that the versioning of items is easy to understand for everyone [Semver](https://semver.org/) versioning is promoted for catalog item versions. Users can assign extra labels to versions if they want to indicate a certain version is not an official release (eg 1.2.3-alpha).
+
```
Comment (SimonV-Skyline) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:22]: Should we here mention the fact an exception is made for protocol which do have their own versioning syntax?
If so, refer to following page: https://docs.dataminer.services/develop/devguide/Connector/ProtocolVersionSemantics.html

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+## How to register a private catalog item
+
+### Register using the Catalog UI
+
+If you use the UI, you can only register a catalog item. There is no UI yet to register a new version, this can only be done by using the API.
+
+Registering a new catalog item can be done by executing the following steps.
+
+1. Go to the [Catalog](https://catalog.dataminer.services). There you will find a button next to the organization selector called         "Register Catalog". Clicking this button will open a popup with a few fields.
```
Comment (SimonV-Skyline) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md]: looks like some spaces need to be cleaned away here.
"Selector called            "Register Catalog".

**Review by SimonV-Skyline — COMMENTED** [review]
couple of small remark, questions

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 3 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:22]: An exception has not been made yet. We will add this later. For now this is what we support.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+To register a new Catalog item, do the following:
+
+1. Go to the [Catalog](https://catalog.dataminer.services).
+
+1. Open the user menu by clicking the user icon on the top right. Here you will find the "Register item" button.
+
+1. Indicate for which organization you want to register the Catalog item. This can be done by using the selection box that will automatically be populated with all the organizations on the dataminer.services platform you have access to.
+
+1. Select the type of the Catalog item. The type will define how the Catalog item will be shown to the users. We also allow filtering on type so users can quickly find what they are looking for when browsing the Catalog module.
```
Comment (KoenAnnys) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:37]: clarify that with type we mean connector, visio, automation script etc and determines the type of solution/integration you made.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+1. Enter the name of the item. This will be the name that will be used to display the item in the Catalog. It needs to be human readable. Make sure you pick a name that makes sense.
+
+1. Once all information has been filled in, click *Register*. This will conclude the registration process for the Catalog item and return the unique identifier for the item in question. This identifier will allow you to register versions for the catalog item using the [API](xref:Register_Catalog_Item#register-using-the-catalog-api).
+
+### Register using the Catalog API
```
Comment (KoenAnnys) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:43]: perhaps add the b2c authenticated call as well that allows to just register a item (without version). the call that is used by the UI atm.

Comment (KoenAnnys) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:43]: is it the intention that skyline users register their item as well as private?

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
+---
+uid: Register_Catalog_Item
+---
+
+# Registering a Catalog item
+
+## What is a Catalog item?
+
+The content available in the Catalog module has always been provided by Skyline Communications. We have been using this module initially to share our connectors with all of our users. In a later stage, we have been adding app packages, Automation scripts and visual overviews. All of these items are called "Catalog items".
```
Comment (jarno-lernou) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:9]: I think we should be consistent if we use app packages or Application packages. Perhaps the first time we reference, we can add a link to the docs page of it.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+
+## Private Catalog items
+
+All of the items that have been provided by Skyline Communications are considered to be public items. They are available for all our users. Note that for some of these items (e.g. Connectors) users might need to buy a license to be able to use them on their system.
+
+Private Catalog items are items that are only available to one or more organizations. This gives users the possibility to develop app packages, Automation scripts or any type of Catalog item and redeploy them within their organization on various systems or share them with other organizations.
+
+## Versioning of Catalog items
+
+Catalog items can hold multiple versions. To make sure that the versioning of items is easy to understand for everyone, [Semver](https://semver.org/) versioning is promoted for Catalog item versions. Users can assign extra labels to versions if they want to indicate that a certain version is not an official release (e.g. 1.2.3-alpha).
```
[the remark this answers is not in the document]
Comment (jarno-lernou) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:21]: Also what about Scripts or Visio's? @SimonV-Skyline, what's your take on this, but I'm wondering how we decide to make use of three or four numbers.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+
+## How to register a private Catalog item
+
+### Register using the Catalog UI
+
+If you use the UI, you can only register a Catalog item. There is no UI yet to register a new version. This can only be done by using the API.
```
Comment (jarno-lernou) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:27]: Give context who can use the 'API' and if we have documentation available.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 5 above)
Comment (jarno-lernou) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:37]: I agree. I would indicate the available list or share some documentation about this.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 6 above)
Comment (jarno-lernou) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:43]: I would say YES for sure. We should try to polish the Catalog as much as possible. A lot of packages are not meant to be public in any way. We should register them under 'Skyline Communications Development'.

I do think some example packages should be public, but it should be clear (e.g. by label) that it should it is an 'example'.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+# [Required]
+# Possible values: 
+#   - AutomationScript: If the catalog item is an Automation Script that can be deployed on a DataMiner System.
+#   - AppPackage: If the catalog item is a Solution Package that can be deployed on a DataMiner System.
+#   - ProtocolPackage: If the catalog item is a Protocol that can be deployed on a DataMiner System.
+#   - Visio: If the catalog item is a Visio that can be deployed on a DataMiner System.
+type: 'AutomationScript'
+
+# [Required] 
+# The id of the catalog item.
```
Comment (jarno-lernou) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:69]: indicate from where this is coming

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+This file will contain all necessary information to register a Catalog item or version. This manifest file should be a valid yml file and will contain "required" and "optional" attributes to add extra information to the Catalog item. Note that limitations may apply to certain attributes based on length or formatting.
+
+```yml
+# [Required]
+# Possible values: 
+#   - AutomationScript: If the catalog item is an Automation Script that can be deployed on a DataMiner System.
+#   - AppPackage: If the catalog item is a Solution Package that can be deployed on a DataMiner System.
+#   - ProtocolPackage: If the catalog item is a Protocol that can be deployed on a DataMiner System.
+#   - Visio: If the catalog item is a Visio that can be deployed on a DataMiner System.
+type: 'AutomationScript'
```
Comment (jarno-lernou) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:66]: Did you check this with Data Acquisition? This is something that is typically also clear from the GitHub Action?

In this way, we can still make use of an Automation Script solution registering as a 'AppPackage' (ref. discussion earlier)? Can you elaborate?

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 12]
```diff
@@ -0,0 +1,139 @@
...
+
+# [Required] 
+# The version in format X.X.X
+# Once a version has been published it can no longer be changed.
+# The version should be incremented to do any changes.
+# If you want to create a package that is not final, you can provide a label at the end: X.X.X-label
+# If a catalog item has a label in the version, then the version is considered to be a pre-release version, and not an official one.
+#   Valid versions range from 0.0.0 to 2147483647.2147483647.2147483647
+#   The label can contain characters a-z0-9 with a max length of 30 characters.
+version: '1.0.0'
```
Comment (jarno-lernou) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:93]: Are we then still supposed to 'tag' a solution? What will happen if you don't change the manifest file, but tag with a new version (or the other way around)?

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 13]
```diff
@@ -0,0 +1,139 @@
...
+
+# [Optional]
+# General information about the catalog item.
+#   Max length: 100.000 characters
+description: |
+  This script allows you to easily add a new leaf switch element.
```
Comment (jarno-lernou) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:99]: Do we need this? Don't we have a 'Readme' file?

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 14]
```diff
@@ -0,0 +1,139 @@
...
+
+# [Optional]
+# A short description for this specific version.
+# This message is shown along side each version in the version overviews in the catalog. 
+#   Max length: 500 characters.
+version-message: 'Initial version. Does not contain anything useful yet.'
```
Comment (jarno-lernou) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:105]: Can you please engage with Data Acquisition here as there is some overlap with version history available in connectors and application packages.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 15]
```diff
@@ -0,0 +1,139 @@
...
+
+# [Optional]
+# A valid URL that points to the home page.
+# Usually, this home page is also the location of the source code.
```
Comment (jarno-lernou) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:109]: Can we somehow automatically use the 'URL' when we 'tag' a catalog item to be released and use that link? It feels counter-productive to specify the link manually.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 16]
```diff
@@ -0,0 +1,139 @@
...
+  - name: 'John Smith'
+    email: 'john@skyline.be'
+    url: 'https://github.com/JohnSmith'
+
+# [Optional]
+# Tags that allow you to categorize your catalog items.
+#   Max length: 50 characters.
+#   Can not contain newlines.
+#   Can not contain leading or trailing whitespace characters.
+tags:
```
Comment (jarno-lernou) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:132]: I'm a little bit worried about the amount of tags will potentionally be introduced. How will you cope with this on the Catalog UI?

**Review by jarno-lernou — CHANGES_REQUESTED** [review]

**Review by jarno-lernou — COMMENTED** [review]
Review Remarks given.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:21]: So at the moment we only support semver :) Maybe I will phrase it more clearly.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 9 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:27]: The documentation is on this page.

Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:27]: I linked it.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 6 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:43]: Please register them under Skyline Communications!!
If they later get published it will say "Skyline Communications Development" as publisher....

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 5 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:37]: Added the list on this page and referenced it

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 11 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:66]: Yes this was checked by with Data Acquisition, they were in all meetings that were held to create this file.
The GitHub action will just pass this file to us.
It will indeed be possible to register an automation script as an app package if you would want to do so.
This type is not defining how something is packaged, it defines the type of the Catalog item as it will be shown on the catalog.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 12 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:93]: What do you mean with tag a solution?
Versioning is controlled by the manifest file and nothing else. If you do not update is you will get an error telling you this version already exists.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 15 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:109]: We could detect this in the github action but note that maybe a user will want to point to external documentation so this is why we allow this.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 14 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:105]: As mentioned before they have been involved from the beginning when creating this file.
What is the overlap you are mentioning?

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 13 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:99]: This is the first iteration.
In a future iteration we will allow the readme file. This is not supported yet.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 16 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:132]: We can opt to only show the first x amount of tags.

Comment (brayunm) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:132]: We can limit this in the first version(s), however, this means we'll need a clear list of which tags are available.
ATM we're not limiting what the users can provide for tag(s)

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 3 above)
Comment (SimonV-Skyline) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:22]: Does this mean connectors are not supported in a first phase?

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (SimonV-Skyline) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md]: yes, I think we all understood that you only support semver atm :)
The question is:
- does that mean you allow users to register connectors using semver? -> I would be against that !
- or does that mean you don't yet support connectors -> then I'm OK but we need to mention it explicitely

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 10 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:78]: Will leave this as is for now. The warning was moved up.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md]: Yes we allow that but we can block that if you want, what would be the downside of a customer registering a connector with semver? Might be perfectly fine?
We do support connectors.

[the remark this answers is not in the document]
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md]: If you think this would be better, we can block connectors for registration until we support the connector versioning as well and enforce that then.

[the remark this answers is not in the document]
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md]: Will enforce x.x.x.x(-label) on the connectors

File: user-guide/Cloud_Platform/CloudAdminApp/Managing_DCP_keys.md [hunk 17]
```diff
@@ -2,34 +2,59 @@
 uid: Managing_DCP_keys
 ---
 
-# Managing dataminer.services keys
+# dataminer.services keys
```
Comment (MariekeGO) [inline user-guide/Cloud_Platform/CloudAdminApp/Managing_DCP_keys.md:5]: I'd prefer to keep the original title here, as the page includes everything from viewing the keys to managing them, and we often put such things under one "managing" header. Otherwise, we'll have to change this title in the TOC as well, and it will stand out like a sore thumb between all the other titles in that section:
image

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 18]
```diff
@@ -0,0 +1,150 @@
+---
+uid: Register_Catalog_Item
+---
+
+# Registering a Catalog item
+
+## About Catalog items
```
Comment (MariekeGO) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:7]: Could we move line 7 to 32 to a separate page, below "About the Catalog module"? That way we separate the concept information from the actual how-to, which will probably be more user-friendly, especially since the concept info could also be relevant for other topics.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 19]
```diff
@@ -0,0 +1,150 @@
...
+> If the item is of type Connector, you will need to adhere to the [existing connector versioning](xref:ProtocolVersionSemantics).
+
+### Supported Catalog item types
+
+At present, the following types of Catalog items are supported:
+
+- Automation Script
+- Application Package
+- Connector
+- Visio
```
Comment (MariekeGO) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:31]: Are these the exact names of the types, or can these be rephrased? This because the casing for Automation Script and Application Package is inconsistent with the rest of the documentation, and we usually also avoid using "Visio" on its own (should be "Visio drawing" or "Visio file" instead).

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 20]
```diff
@@ -0,0 +1,150 @@
...
+
+If you use the UI, you can only register a Catalog item. There is no UI yet to register a new version. This can only be done by using the [API](xref:Register_Catalog_Item#register-using-the-catalog-api).
+
+To register a new Catalog item, do the following:
+
+1. Go to the [Catalog](https://catalog.dataminer.services).
+
+1. Open the user menu by clicking the user icon in the top-right corner.
+
+   Here you will find the *Register item* button.
```
Comment (MariekeGO) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:45]: Should users click this button? If so, this should be included as a step of its own, instead of an indented line.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 18 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:7]: Do we need to add this to the "About the Catalog module" page instead?
Do not really know what title we would give the page otherwise.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 3 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:23]: They are, we will enforce the connector versioning.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 19 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:31]: What would be the correct casing/naming? These are not the exact namings, the exact namings are added at the top of the YAML example.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 20 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:45]: done

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 18 above)
Comment (MariekeGO) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:7]: Yes, that's also a valid option. Then we can create a subsection "Accessing the Catalog module" with the info that's already there, and "About Catalog items" with this info.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md (hunk 19 above)
Comment (MariekeGO) [reply user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:31]:
```suggestion
- Automation script
- Application package
- Connector
- Visio file
```

File: user-guide/Cloud_Platform/CloudAdminApp/Managing_DCP_keys.md (hunk 17 above)
Comment (baptistepa) [reply user-guide/Cloud_Platform/CloudAdminApp/Managing_DCP_keys.md:5]: done

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 21]
```diff
@@ -0,0 +1,133 @@
...
+
+1. Click the *Register item* button.
+
+1. Select the organization for which you want to register the Catalog item.
+
+   The selection box is automatically populated with all the organizations you have access to on the dataminer.services platform.
+
+1. Select the type of the Catalog item.
+
+   This can be one of the supported [Catalog item types](xref:About_the_Catalog_module#supported-catalog-item-types). The type will define how the Catalog item will be shown to the users. Users can also filter on type to quickly find what they are looking for when browsing the Catalog.
```
Comment (MariekeGO) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:27]: If only the supported catalog item types can be selected, I'd rephrase this as follows to prevent possible confusion:
```suggestion
   The selected [Catalog item type](xref:About_the_Catalog_module#supported-catalog-item-types) defines how the Catalog item will be shown to the users. Users can also filter on type to quickly find what they are looking for when browsing the Catalog.
```

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 22]
```diff
@@ -0,0 +1,133 @@
...
+This file will contain all necessary information to register a Catalog item or version. This manifest file should be a valid .yml file and will contain "required" and "optional" attributes to add extra information to the Catalog item. Note that limitations may apply to certain attributes based on length or formatting.
+
+> [!IMPORTANT]
+> If the *id* field is not filled in, a new Catalog item registration will be created. If you want to register a new version for an existing Catalog item, make sure to fill in the ID of that item.
+
+```yml
+# [Required]
+# Possible values: 
+#   - AutomationScript: If the catalog item is an Automation Script that can be deployed on a DataMiner System.
+#   - AppPackage: If the catalog item is a Solution Package that can be deployed on a DataMiner System.
```
Comment (KoenAnnys) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:60]: values should be as below (appPackage, protocolpackage is reference to package format but not catalog type): 

image

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 23]
```diff
@@ -0,0 +1,133 @@
...
+
+### Authentication
+
+The API call is authenticated using [organization keys](xref:Managing_DCP_keys#organization-keys). The key needs to be added to the HTTP request in a header called **Ocp-Apim-Subscription-Key**.
+
+### API call signature
+
+The call to register a Catalog item can be found at the following URL: <https://api.dataminer.serivces/api/key-catalog/v1-0/catalog/register>. To call the API you will need to use the **POST** HTTP method.
+
+The call accepts a binary file as input. This file should be a zip file containing a ["manifest.yml"](xref:Register_Catalog_Item#manifest-file) and the item you want to upload.
```
Comment (KoenAnnys) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:47]: best to mention that it needs to be Content/Type "multipart/form-data" of type file where key is "file" and value the zip file.

File: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md [hunk 24]
```diff
@@ -0,0 +1,133 @@
...
+> [!IMPORTANT]
+> If the *id* field is not filled in, a new Catalog item registration will be created. If you want to register a new version for an existing Catalog item, make sure to fill in the ID of that item.
+
+```yml
+# [Required]
+# Possible values: 
+#   - automationscript: If the catalog item is an Automation Script that can be deployed on a DataMiner System.
+#   - package: If the catalog item is a Solution Package that can be deployed on a DataMiner System.
+#   - connector: If the catalog item is a Protocol that can be deployed on a DataMiner System.
+#   - visio: If the catalog item is a Visio that can be deployed on a DataMiner System.
```
Comment (MariekeGO) [inline user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md:62]:
```suggestion
#   - visio: If the catalog item is a Visio file that can be deployed on a DataMiner System.
```

**Comment by baptistepa** [comment]
@MariekeGO this can be merged :) (finally)

**Review by MariekeGO — APPROVED** [review]
